.gitignore

```diff
@@ -1,2 +1 @@
 /node_modules
```

Comment:
Should the `build` folder also be excluded?

Reply:
Yes, it would be better to add `/build` here. As long as no one commits it, there's no harm:
```
$ git status
On branch decaffeinate
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	build/
```

Reply:
Done 👍